examples/petsc/src/matops.c

```diff
@@ -73,7 +70,6 @@ PetscErrorCode MatGetDiag(Mat A, Vec D) {
 // This function uses libCEED to compute the action of the Laplacian with Dirichlet boundary conditions
 // -----------------------------------------------------------------------------
 PetscErrorCode ApplyLocal_Ceed(Vec X, Vec Y, OperatorApplyContext op_apply_ctx) {
-  PetscScalar *x, *y;
   PetscMemType x_mem_type, y_mem_type;
 
   PetscFunctionBeginUser;
```

Comment:
Probably on the ToDo list should go making these `PetscFunctionBeginUser` and `PetscFunctionReturn` match the PETSc whitespace conventions (no newline after Begin or before Return)